contracts/RelayHubApi.sol

```diff
@@ -1,6 +1,6 @@
 pragma solidity >=0.4.0 <0.6.0;
 
-contract  RelayHubApi {
+contract RelayHubApi {
```

Comment:
better use the interface notation `IRelayHub`

Reply:
This one was done in a previous pull request... 
It was called  RelayHubApi to be consistent with RelayRecipientApi. I can rename them but not sure it if it matters

Reply:
done